state-chain/pallets/cf-asset-balances/src/tests.rs

```diff
@@ -180,16 +192,23 @@ pub fn refund_validators_polkadot() {
 #[test]
 pub fn max_refunds_per_epoch() {
 	new_test_ext().execute_with(|| {
-		for i in 0..crate::MAX_REFUNDED_VALIDATORS_ETH_PER_EPOCH + 2 {
+		for i in 0..crate::MAX_REFUNDED_VALIDATORS_ETH_PER_EPOCH {
 			payed_gas(
 				ForeignChain::Ethereum,
 				100,
 				ForeignChainAddress::Eth(sp_core::H160([i as u8; 20])),
 			);
 		}
+		for i in 254u8..=255u8 {
+			payed_gas(
+				ForeignChain::Ethereum,
+				500,
```

Comment:
I'm not sure what the benefit of this is? We don't test anything related to the amount.

Reply:
Oh this is to test that the payouts are settled in reverse order (smallest to largest) so the 2 largest fees are left when we run out of available fund. I'll add a comment for this.